src/vm/builtins/builtin_runner/bitwise.zig

```diff
@@ -24,6 +44,26 @@ pub const BitwiseBuiltinRunner = struct {
     /// Number of instance per component
     instances_per_component: u32,
 
+    /// Retrieve the felt in memory that an address denotes as an integer within the configured `total_n_bit` limit,
+    /// default is 251
+    /// # Arguments
+    /// - address: The address belonging to the Bitwise builtin's segment
+    /// - memory: The cairo memory where addresses are looked up
+    /// # Returns
+    /// The felt as an integer.
+    fn getIntWithinBits(self: Self, address: Relocatable, memory: *Memory) BitwiseError!u256 {
```

Comment:
why not using a pointer as first arg? like 

```zig
fn getIntWithinBits(self: *Self, address: Relocatable, memory: *Memory)
```

Reply:
you'd have a pointer to self when you want to mutate state, right? lmk if im understanding